.github/workflows/build.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           git-user: ${{ env.GITHUB_ACTOR }}
 
       - name: Run integration tests
-        run: cargo test --no-default-features --features contract --test contract
+        run: cargo test --test contract
```

Comment:
Why was this removed? I explicitly set it this way so that we wouldnt send telemetry from CI runs?

Reply:
Perhaps it needs a comment above each so it doesnt get restored again in the future.